openfl-docker/Dockerfile.base

```diff
@@ -1,122 +1,33 @@
-# Copyright (C) 2020-2023 Intel Corporation
+# Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
+# ------------------------------------
+# OpenFL Base Image
+# ------------------------------------
+FROM ubuntu:22.04 as base
 
-# If your machine is behind a proxy, make sure you set it up in ~/.docker/config.json
```

Comment:
IMO we need to keep this line unless you've stated it somewhere else

Reply:
Done.